s3file/static/s3file/js/s3file.js

```diff
@@ -147,11 +153,7 @@
     forms.forEach(function (form) {
       form.addEventListener('submit', function (e) {
         e.preventDefault()
-        uploadS3Inputs(e.target)
-      })
-      var submitButtons = form.querySelectorAll('input[type=submit], button[type=submit]')
-      Array.from(submitButtons).forEach(function (submitButton) {
-        submitButton.addEventListener('click', clickSubmit)
+        uploadS3Inputs(e.target, e.submitter)
```

Comment:
Since this boiled down to a line, maybe it's good to pass the entire event aka. make this method the event listener.